### ISSUE TITLE
zdharma/zinit repository does not exist

### ISSUE THREAD

**Issue by jsfelix** [zdharma-continuum/zinit#6]
**Describe the bug**
zdharma/zinit repository does not exist when try to install zinit from `doc/install.sh`

**To Reproduce**
```bash
sh -c "$(curl -fsSL https://raw.githubusercontent.com/zdharma-continuum/zinit/master/doc/install.sh)"
```

**Expected behavior**
install zinit

**Versions:**
zsh 5.8 (x86_64-apple-darwin21.0)

**Comment by alichtman** [comment]
Can you paste the error output here?

**Comment by alichtman** [comment]
Should be fixed in 3308ff8f.
